src/main/java/unrc/dose/BellyService.java

```diff
@@ -0,0 +1,63 @@
+package unrc.dose;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicInteger;
+
+/** This is a Service to store a bunch of Bellies together. */
+public final class BellyService {
+
+    /** Bellies store. */
+    private static Map<String, Belly> bellies = new HashMap<String, Belly>();
```

Comment:
If there would only be one instance of `BellyService` then the class should follow a `Singleton` pattern and this field should not be declared as `static`, but if otherwise there can be multiple instances of this class then shouldn't each service have a map of their own?